org.skytemple.SkyTemple.yml

```diff
@@ -0,0 +1,144 @@
+sdk: org.gnome.Sdk
+command: /app/bin/run.sh
+finish-args:
+  - "--share=network"
+  - "--socket=fallback-x11"
+  - "--socket=wayland"
+  - "--filesystem=host"
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.rust-nightly
+cleanup:  # todo
+  - /bin/pcap-config
+  - /include
+  - /lib/libpcap.a
+  - /lib/pkgconfig
+  - /share/man
+modules:
+  - name: armips
+    buildsystem: simple
+    build-commands:
+      - mkdir -p build
+      - patch Core/ExpressionFunctions.cpp < armips.patch
+      - cd build && cmake -DCMAKE_BUILD_TYPE=Release ..
+      - cd build && make armips-bin
+      - install -Dm755 build/armips /app/bin/armips
+    sources:
+      - type: archive
+        url: https://github.com/Kingcom/armips/archive/v0.11.0.tar.gz
+        sha256: c94e29dfda3bdf853590d825562b9c73a3d8e8e995555e021c6b2a6451572681
+      - type: file
+        path: patches/armips.patch
+  - name: skytemple
+    buildsystem: simple
+    build-commands:
+      - pip3 install --prefix=/app --no-deps  .
+      # Icons
+      - mkdir -p /app/share/icons/hicolor/16x16/apps
+      - install -Dm644 skytemple/data/icons/hicolor/16x16/apps/skytemple.png /app/share/icons/hicolor/16x16/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/32x32/apps
+      - install -Dm644 skytemple/data/icons/hicolor/32x32/apps/skytemple.png /app/share/icons/hicolor/32x32/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/64x64/apps
+      - install -Dm644 skytemple/data/icons/hicolor/64x64/apps/skytemple.png /app/share/icons/hicolor/64x64/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/128x128/apps
+      - install -Dm644 skytemple/data/icons/hicolor/128x128/apps/skytemple.png /app/share/icons/hicolor/128x128/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/256x256/apps
+      - install -Dm644 skytemple/data/icons/hicolor/256x256/apps/skytemple.png /app/share/icons/hicolor/256x256/apps/org.skytemple.SkyTemple.png
+      - mkdir -p /app/share/icons/hicolor/512x512/apps
+      - install -Dm644 skytemple/data/icons/hicolor/512x512/apps/skytemple.png /app/share/icons/hicolor/512x512/apps/org.skytemple.SkyTemple.png
+      # Runner
+      - install -Dm755 run.sh /app/bin/run.sh
+    sources:
+      - type: archive
+        url: https://github.com/SkyTemple/skytemple/archive/refs/tags/1.3.11.post1.tar.gz
+        sha256: cef04b4330da4b6e89f595af3ba8b5d8962717a9dc01fc7e4d0dd73917827e32
+      - type: file
+        path: run.sh
+    modules:
+      # This needs to be done seperately due to some patches and the Rust compiler and Cargo packages required.
+      # Also this needs build isolation, unlike the rest.
+      - name: python3-skytemple-rust
+        buildsystem: simple
+        build-commands:
+          - patch skytemple-rust-1.3.7/setup.py < skytemple_rust_setup_py.patch
+          - tar -czf skytemple-rust-1.3.7.tar.gz skytemple-rust-1.3.7 && rm -rf skytemple-rust-1.3.7
```

Comment:
why don't you just download the tarball?

also pip can install from directories.

Reply:
and just to add, patches are handled properly too. Just make them valid (`diff -u` or `git diff` or `git show` do it right)

Reply:
There is a comment down in the sources on why I don't download the tarball. Pip didn't want to accept the directory for the offline installation.

Reply:
it's about the patches. See my comment about the patches.

Reply:
also you can `pip install .`

Reply:
Oh derp, right of course.